pkg/kubectl/apps/BUILD

```diff
@@ -10,7 +10,20 @@ go_library(
     name = "go_default_library",
     srcs = ["kind_visitor.go"],
     importpath = "k8s.io/kubernetes/pkg/kubectl/apps",
-    deps = ["//vendor/k8s.io/apimachinery/pkg/runtime/schema:go_default_library"],
+    deps = [
+        "//vendor/k8s.io/apimachinery/pkg/runtime/schema:go_default_library",
+    ],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = ["kind_visitor_test.go"],
+    importpath = "k8s.io/kubernetes/pkg/kubectl/apps",
```

Comment:
This is wrong.
It should be `k8s.io/kubernetes/pkg/kubectl/apps_test`.

Don't delete the whole BUILD file and regenerate it.